README.md

```diff
@@ -1,4 +1,28 @@
-# NYSURE application
+#                                              _**NYSURE - The new insurance solution**_
+
+### Nysure is an application that helps end users gather their insurance information, get quotes that compare several insurance companies, and send claims directly to the appropriate insurance company. We work hand in hand with companies aiming at using new technologies to their full potential, in order to make their users’ experiences more effective, pleasant, and simple. 
```

Comment:
Nysure is an application that helps people gather all their different insurance information in one single place. The difference from other solutions out there is that Nysure prioritizes the items you want to keep safe rather than just the policies you posses. For example, if a person purchases a new item (i.e a new mobile phone), they can add the item, include the date of purchase, cost and even photos of the item itself regardless of having an insurance that will cover it. With the quote function inside Nysure, the user can get quotes that compare several companies, and send claims directly to the appropriate insurance company....